openapi/nexus.json

```diff
@@ -12768,34 +12768,34 @@
             "description": "Sets the user's password to the provided value",
             "type": "object",
             "properties": {
-              "details": {
-                "$ref": "#/components/schemas/Password"
-              },
-              "user_password_value": {
+              "mode": {
                 "type": "string",
                 "enum": [
                   "password"
                 ]
+              },
+              "value": {
+                "$ref": "#/components/schemas/Password"
               }
             },
             "required": [
-              "details",
-              "user_password_value"
+              "mode",
+              "value"
             ]
           },
           {
             "description": "Invalidates any current password (disabling password authentication)",
             "type": "object",
             "properties": {
-              "user_password_value": {
+              "mode": {
                 "type": "string",
                 "enum": [
                   "invalid_password"
```

Comment:
Perhaps this should be `invalidate_password` instead? `invalid_password` implies a password that's not valid, not something that will remove passwords.

Reply:
Or maybe even "no_login" or "login_disallowed"?